.azure-pipelines-templates/matrix.yml

```diff
@@ -15,7 +15,7 @@ parameters:
 
   build:
     common:
-      cmake_args: '-DCMAKE_C_COMPILER_LAUNCHER="ccache" -DCMAKE_CXX_COMPILER_LAUNCHER="ccache" -DENABLE_2TX_RECONFIG=ON'
+      cmake_args: '-DCMAKE_C_COMPILER_LAUNCHER="ccache" -DCMAKE_CXX_COMPILER_LAUNCHER="ccache" -DENABLE_2TX_RECONFIG=OFF'
```

Comment:
Perhaps remove `-DENABLE_2TX_RECONFIG` altogether as the default is `OFF` anyway?